ams/ams-server/src/main/java/com/netease/arctic/ams/server/ArcticMetaStore.java

```diff
@@ -369,6 +374,7 @@ private static void checkLeader() {
           Thread.sleep(checkLeaderInterval);
         } catch (InterruptedException e) {
           LOG.warn("notLeader thread was interrupted: " + e.getMessage());
+          return;
```

Comment:
Should the checkLeader Thread be interrupted and exit after ams stopped?

Reply:
add while (isLeader.get())